test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js

```diff
@@ -0,0 +1,307 @@
+
+          cy.get('input[type="radio"]')
+            .siblings('label')
+            .contains('Yes, these organisation details match')
+            .prev('input[type="radio"]')
+            .check()
+
+          cy.get('#confirm-org-details-form button[type="submit"]').click()
+          cy.title().should('eq', 'Settings - Stripe details - GOV.UK Pay')
+          cy.get('h1').should('contain', 'Stripe details')
+          cy.location('pathname').should('not.contain', '/organisation-details/index')
+          cy.get('.govuk-task-list__item')
+            .contains('Confirm your organisation\'s name and address match your government entity document')
+            .parent()
+            .parent()
+            .within(() => {
+              cy.get('.govuk-task-list__status').should('contain.text', 'Complete')
+            })
+        })
+      })
+      describe('When selecting no', () => {
+        describe('When submitting valid organisation details', () => {
+          beforeEach(() => {
+            setStubs({}, [
+              ...Array(3).fill(stripeAccountSetupStubs.getStripeSetupProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE
+              })),
+              gatewayAccountStubs.getStripeAccountByServiceIdAndAccountType(
+                SERVICE_EXTERNAL_ID,
+                LIVE_ACCOUNT_TYPE,
+                {
+                  stripeAccountId: STRIPE_ACCOUNT_ID
+                }
+              ),
+              stripePspStubs.updateAccount({
+                stripeAccountId: STRIPE_ACCOUNT_ID
+              }),
+              stripeAccountSetupStubs.patchStripeProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE
+              }),
+              serviceStubs.patchUpdateServiceGatewayAccounts({
+                serviceExternalId: SERVICE_EXTERNAL_ID
+              }),
+              stripeAccountSetupStubs.getStripeSetupProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE,
+                organisationDetails: true
+              })
+            ])
+            cy.visit(STRIPE_DETAILS_SETTINGS_URL + '/organisation-details/index')
+          })
+
+          it('should redirect to the task summary page on success', () => {
+            cy.get('input[type="radio"]')
+              .siblings('label')
+              .contains('No, these organisation details do not match')
+              .prev('input[type="radio"]')
+              .check()
+
+            cy.get('#confirm-org-details-form button[type="submit"]').click()
+            cy.location('pathname').should('contain', '/organisation-details/update')
+            cy.get('h1').should('contain', 'What is the name and address of your organisation on your government entity document?')
```

Comment:
A screenshot for this in the PR would be great.

Reply:
screens of the task view? they were in a previous PR: https://github.com/alphagov/pay-selfservice/pull/4361